src/main/resources/db/changelog/changes/v1.2.0/update-drv-purchase-order-line-details-definition.xml

```diff
@@ -0,0 +1,525 @@
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+  <changeSet id="update-purchase-order-line-details-entity-type-definition" runOnChange="true"
+    author="bsharp@ebsco.com,novercash@ebsco.com,kjain@ebsco.com">
+    <update tableName="entity_type_definition">
+      <column name="definition">
+        {
+          "id": "90403847-8c47-4f58-b117-9a807b052808",
+          "name": "drv_purchase_order_line_details",
+          "private": false,
+          "fromClause": "src_purchase_order_line JOIN src_purchase_order purchase_order ON purchase_order.id = ((src_purchase_order_line.jsonb ->> 'purchaseOrderId'::text)::uuid) LEFT JOIN src_users_users user_details ON user_details.id = (((purchase_order.jsonb -> 'metadata'::text) ->> 'createdByUserId'::text)::uuid) LEFT JOIN src_users_users user_details_for_order_updated_by ON user_details_for_order_updated_by.id = (((purchase_order.jsonb -> 'metadata'::text) ->> 'updatedByUserId'::text)::uuid) LEFT JOIN src_users_users user_details_for_pol_created_by ON user_details_for_pol_created_by.id = (((src_purchase_order_line.jsonb -> 'metadata'::text) ->> 'createdByUserId'::text)::uuid) LEFT JOIN src_users_users user_details_for_pol_updated_by ON user_details_for_pol_updated_by.id = (((src_purchase_order_line.jsonb -> 'metadata'::text) ->> 'updatedByUserId'::text)::uuid) LEFT JOIN src_organizations organization_details ON organization_details.id = ((purchase_order.jsonb ->> 'vendor'::text)::uuid) LEFT JOIN src_users_users user_details_of_assignee ON user_details_of_assignee.id = ((purchase_order.jsonb ->> 'assignedTo'::text)::uuid) ",
+          "columns": [
+            {
+              "name": "acqunit_ids",
+              "dataType": {
+                "dataType": "arrayType",
+                "itemDataType": {
+                  "dataType": "rangedUUIDType"
+                }
+              },
+              "queryable": false,
+              "valueGetter": "( SELECT array_agg(acq_id.value::text) FILTER (WHERE (acq_id.value::text) IS NOT NULL) AS array_agg FROM jsonb_array_elements_text(purchase_order.jsonb -> 'acqUnitIds'::text) acq_id(value))",
+              "filterValueGetter": "( SELECT array_agg(lower(acq_id.value::text)) FILTER (WHERE (acq_id.value::text) IS NOT NULL) AS array_agg FROM jsonb_array_elements_text(purchase_order.jsonb -> 'acqUnitIds'::text) acq_id(value))",
+              "valueFunction": "lower(:value)",
+              "visibleByDefault": false,
+              "source": {
```

Comment:
Added API here

Reply:
I gotta check the obvious thing, just because it's something that I keep messing up when I add value source API settings 🙂: do we need to add these interfaces and/or their associated permissions to the module descriptor?

Reply:
This `source` property looks like it's pointing at a different entity type. For value source APIs to work, we need the `source` to point to this entity type and field. `source` configures how the UI should request the available values from the BE, so it should only point to a different ET+field when that different ET+field are where the available values are defined.

```json
              "source": {
                "entityTypeId": "90403847-8c47-4f58-b117-9a807b052808",
                "columnName": "acqunit_ids"
              },
```